CHANGES.rst

```diff
@@ -29,6 +29,11 @@ Infrastructure, Utility and Other Changes and Additions
   The API for JPLspec's ``lookup_table.find`` function returns a dictionary
   instead of values (for compatibility w/CDMS).  [#2144]
 
+- The functions ``astroquery.utils.decorators.deprecated()`` and
```

Comment:
These never was part of the public API and was only internally used, so won't need a changelog.

https://astroquery.readthedocs.io/en/latest/utils.html?highlight=decorator